test/functional/p2p_compactblocks_hb.py

```diff
@@ -17,9 +17,9 @@ def set_test_params(self):
 
     def peer_info(self, from_node, to_node):
         """Query from_node for its getpeerinfo about to_node."""
-        for peerinfo in self.nodes[from_node].getpeerinfo():
-            if "(testnode%i)" % to_node in peerinfo['subver']:
-                return peerinfo
+        for peer in self.nodes[from_node].getpeerinfo():
```

Comment:
In 52e5ce2762cb025a8d7d6b61e1121f9a09367689:
The changes to the tests to tolerate a missing `subver` should no-longer be needed now that these fields are non-optional?
Commit message also mentions marking the field optional, but that no-longer happens (and doesn't need to).

Reply:
Right, done.